scripts/us_census/acs5yr/subject_tables/process.py

```diff
@@ -50,14 +54,18 @@
     '[for processing]set the flag to add additional columns to debug')
 
 
-def set_column_map(input_path, spec_path, output_dir):
+def set_column_map(input_path, spec_path, output_dir, create_mcf):
     generated_col_map = process_zip_file(input_path,
                                          spec_path,
                                          write_output=False)
+    output_dir = os.path.expanduser(output_dir)
     f = open(os.path.join(output_dir, 'column_map.json'), 'w')
     json.dump(generated_col_map, f, indent=4)
     f.close()
 
+    if create_mcf:
+        generate_mcf_from_column_map(generated_col_map, output_dir)
```

Comment:
Since this method is called by the option you leave `create_mcf` boolean to be True by default. Also once you bring the `generate_mcf_from_column_map` inside process.py you can call it directly after line 60.

Reply:
it is True by default, have added this section in case user doesn't want the mcf file to be generated.